<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepTootController</class>
 <widget class="QMainWindow" name="DeepTootController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="InitialConditionsGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>491</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Initial Conditions</string>
    </property>
    <widget class="QLabel" name="label_initialConditionPresets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial Condition Preset</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_initialConditionPresets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Ball Towards Goal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ball Away from Goal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scenario Preset 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scenario Preset 2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ControllerSetupGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Controller Setup</string>
    </property>
    <widget class="QGroupBox" name="ControllerParametersGroup">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Controller Parameters</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="MiscOptionsGroup">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Misc Options</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_drivingControllerType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Feed Forward and Feed Back</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Feed Foward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PID Feedback Controller</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stanley Controller</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_drivingControllerType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Driving Controller Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_aerialControllerType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Feed Foward and Feed Back</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P2 Controller</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Feed Forward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_aerialControllerType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Aerial Controller Type</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_debuggingOptions">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Debugging Options</string>
    </property>
    <widget class="QPushButton" name="pushButton_startDebugging">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Debugging</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_debuggingType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Aerial Only Debugging</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Driving Only Debugging</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug Type 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug Type 2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_plottingOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Plotting Options</string>
    </property>
    <widget class="QPushButton" name="pushButton_plotData">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot Data</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_carPosition">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_carVelocity">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_carOrientation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dataToPlot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data to Plot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ballPosition">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ballVelocity">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_boost">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Boost</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_inputThrottle">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Throttle</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_inputSteer">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Steer</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_carAngVel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Angular Velocity</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_inputJump">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Jump</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectCarData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Car Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectInputData">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_selectBallData">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ball Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_execute">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>810</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ScenarioParametersGroup_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Trajectory Generator</string>
    </property>
    <widget class="QLabel" name="label_trajectoryGenerationType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trajectory Generator Type</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="BrainParametersGroup">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Generation Parameters</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="FinalConditionsGroup_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Other Options</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_scenarioPresets_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Guided Action Type 1 (Minimum Time to Ball)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Guided Action Type 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neural Network 2</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_guided">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Guided</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
